inc/class-core-sitemaps.php

```diff
@@ -57,32 +57,35 @@ public function register_sitemaps() {
 		/**
 		 * Filters the list of registered sitemap providers.
 		 *
-		 * @since 0.1.0
-		 *
 		 * @param array $providers Array of Core_Sitemap_Provider objects.
+		 * @since 0.1.0
 		 */
-		$providers = apply_filters( 'core_sitemaps_register_providers', array(
-			'posts'      => new Core_Sitemaps_Posts(),
-			'pages'      => new Core_Sitemaps_Pages(),
-			'categories' => new Core_Sitemaps_Categories(),
-			'users'      => new Core_Sitemaps_Users(),
-		) );
+		$providers = apply_filters(
+			'core_sitemaps_register_providers',
+			array(
+				'posts'      => new Core_Sitemaps_Posts(),
+				'pages'      => new Core_Sitemaps_Pages(),
+				'categories' => new Core_Sitemaps_Categories(),
+				'users'      => new Core_Sitemaps_Users(),
+			)
+		);
 
 		// Register each supported provider.
 		foreach ( $providers as $provider ) {
-			$this->registry->add_sitemap( $provider->name, $provider );
+			$this->registry->add_sitemap( $provider->slug, $provider );
```

Comment:
required change as $name is no longer in use (not compatible with multiple sitemaps per provider)

Reply:
Undone.